invenio_records_resources/resources/files/resource.py

```diff
@@ -84,13 +85,16 @@ def create_url_rules(self):
             ]
         return url_rules
 
+    @request_extra_args
```

Comment:
any reason why you chose the extra_args instead of view args ? I did the opposite actually in rdm-records and I am considering changing